python/PresentationBuilder/slides/RemarkSlide.py

```diff
@@ -190,6 +189,19 @@ def parse(self, markdown):
       if self.isParamValid(key):
           markdown = key + ':' + self.parameters()[key] + '\n' + markdown
 
+    # Adjust code starting with *
+    # Remark automatically highlights code lines starting with *, this is disabled
+    # by removing the background in the css and adding an extract *, I hope Remark
+    # will get updated so this is not necessary.
+    #
+    # Also, for some reason re.sub doesn't like this re, so I am doing it manually
+    regex = r'```(.*?)```'
+    for m in re.finditer(regex, markdown, re.DOTALL | re.MULTILINE):
+      for line in m.group(1).split('\n'):
+        if line.strip().startswith('*'):
+          print line
```

Comment:
You don't need the sub() version with a function. This method just prepends a '*' so you can just do that with the string version of sub().